nik4.py

```diff
@@ -231,6 +246,13 @@ def add_fonts(path):
 		fmt = 'png256'
 	
 	need_cairo = fmt in ['svg', 'pdf']
+
+	# output projection
+	if options.projection.lower().startswith('epsg:'):
+		proj_target = mapnik.Projection('+init={}'.format(options.projection.lower()))
```

Comment:
If the code understands epsg numbers, why not initialize projections above with `epsg:3857` and `epsg:4326`?

Reply:
I don't understand what you mean. If `--projection epsg:1234` was used, I initialise `mapnik.Projection` with `+init=epsg:1234`. But if a Proj4 string was specified, I have to initialise `mapnik.Projection` with the string. It is not possible to just use `mapnik.Projection("epsg:4326")`. You have to prefix it with `+init=` but I decided not to force users to always type `+init=` if they want to use `--projection`.

Reply:
Okay, let's leave that as is.